src/inference/mh.jl

```diff
@@ -90,7 +90,11 @@ function set_namedtuple!(vi::VarInfo, nt::NamedTuple)
             vi[vns[1]] = vals
         elseif v_isarr && n_vals == n_vns > 1
             for (vn, val) in zip(vns, vals)
-                vi[vn] = [val]
+                if val isa AbstractArray
+                    vi[vn] = val
+                else
+                    vi[vn] = [val]
+                end
```

Comment:
I always write that as vi[vn] = [val;]. But it's annoying that you have to do it at all...